includes/Interfaces/Product_Query.php

```diff
@@ -39,7 +39,8 @@ interface Product_Query {
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term.
+	 * @param string $sort_by sort order for query.
 	 * @return Product[]|WP_Error
 	 */
-	public function get_search( string $search_term );
+	public function get_search( string $search_term, string $sort_by);
```

Comment:
As per https://github.com/GoogleForCreators/web-stories-wp/pull/11552/files#r880339767, this becomes much easier with separate `$orderby` and `$order` params